### ISSUE TITLE
`oy_read()` should check that CSVs are journey history files

### ISSUE THREAD

**Issue by matt-dray** [matt-dray/oystr#3]
Currently the function reads the CSVs and binds them all.

```
do.call("rbind", lapply(oy_files, function(x) read.csv())
```

Probably want to read the files, then check content (perhaps dimensions, headers), then bind the ones that are the correct format.

**Comment by matt-dray** [comment]
Very simple solution for now is `Filter(function(x) length(x) == 8, oy_list)`, since the TfL journey history statements have exactly 8 column. Would be good to match the column names to expected names.

**Comment by matt-dray** [comment]
Closed with 697a45e6577d9ead1c5c0f0fd59c633bd691cb3e

Now has:

```
suppressWarnings(
  element_logical <- sapply(
    X = names(data_list_all),
    FUN = function(x) all(names(data_list_all[[x]]) == c(
      "Date", "Start.Time", "End.Time", "Journey.Action",
       "Charge", "Credit", "Balance", "Note"
     )
     )
  )
)

data_list_subset <- data_list_all[element_logical]
```
